apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
```

Comment:
fair enough I shall change it 
piggyback? duuude, have you seen the size of me compared to you?

![alt text](http://gph.is/295a1cA)